lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -936,16 +940,29 @@ fixtureWallet ctx = do
             then return (getFromResponse id r)
             else threadDelay oneSecond *> checkBalance w
 
--- | Restore a Byron faucet and wait until funds are available.
+-- | Restore a Byron faucet wallet with a default name and passphrase.
 fixtureByronWallet
     :: Context t
     -> IO ApiByronWallet
-fixtureByronWallet ctx = do
+fixtureByronWallet =
+    fixtureByronWalletWith
+        "Faucet Byron Wallet"
+        "Secure Passphrase"
+
+-- | Restore a Byron faucet wallet with the specified name and passphrase.
+fixtureByronWalletWith
+    :: Text
+        -- ^ name
+    -> Text
+        -- ^ passphrase
+    -> Context t
+    -> IO ApiByronWallet
+fixtureByronWalletWith name passphrase ctx = do
```

Comment:
I just noticed, but this is rather different from the its dual `fixtureWalletWith` 

https://input-output-hk.github.io/cardano-wallet/haddock/edge/cardano-wallet-core-integration-2019.6.24/Test-Integration-Framework-DSL.html#v:fixtureWalletWith

Plus, I don't see much the benefits of defining your own name and passphrase on a fixture wallet?

Reply:
> Plus, I don't see much the benefits of defining your own name and passphrase on a fixture wallet?

From the point of view of the test, it doesn't know the passphrase that `byronFixtureWallet` happens to use, as that's an implementation detail.

But the test _does_ need to send a passphrase as a payload when calling `migrateByronWallet`.

Allowing the test to define and pass its own passphrase to `fixtureByronWalletWith` allows it to reuse that _same_ passphrase later on when calling `migrateByronWallet`. This means we don't need to repeat the magic constant encoded in `fixtureByronWallet`.

Reply:
> From the point of view of the test, it doesn't know the passphrase that fixtureByronWallet happens to use, as that's an implementation detail.

It's more of a constant of the test framework, although we should really make it a constant and have it exported as `fixtureWalletPassphrase` :)

Reply:
> we should really make it a constant and have it exported as fixtureWalletPassphrase :)

I agree that would be a much better alternative to repeatedly hard-coding the same constant in multiple places. :)